src/main/java/reposense/model/RepoLocation.java

```diff
@@ -17,7 +15,7 @@
 public class RepoLocation {
     private static final String GIT_LINK_SUFFIX = ".git";
```

Comment:
Is this attribute unused after the change?

Reply:
There is still one usage here, which I have not touched.

https://github.com/reposense/RepoSense/blob/60de6237039041e7b2dfe9af208d3775fa61fb51/src/main/java/reposense/model/RepoLocation.java#L38

Reply:
OK